pyke/lightcurve.py

```diff
@@ -134,6 +134,23 @@ def fold(self, period, phase=0.):
             return LightCurve(fold_time[sorted_args], self.flux[sorted_args])
         return LightCurve(fold_time[sorted_args], self.flux[sorted_args], flux_err=self.flux_err[sorted_args])
 
+    def normalize(self):
+        """Returns a normalized version of the lightcurve.
+
+        The normalized lightcurve is obtained by dividing `flux` and `flux_err`
+        by the median flux.
+
+        Returns
+        -------
+        normalized_lightcurve : LightCurve object
+            A new ``LightCurve`` in which the flux is normalized to 1.
```

Comment:
nitpick: maybe replace  ``normalized to 1.`` with ``normalized by the median flux.`` just to be more precise?